runtime-config-linux.md

```diff
@@ -503,3 +503,14 @@ Its value is either slave, private, or shared.
 ```json
     "rootfsPropagation": "slave",
 ```
+
+## No new privileges
+
+Setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
```

Comment:
Why not?

On Wed, Jan 20, 2016 at 2:14 PM, Vincent Batts <notifications@github.com>
wrote:

> In runtime-config-linux.md
> <https://github.com/opencontainers/specs/pull/290#discussion_r50328465>:
>
> > @@ -503,3 +503,14 @@ Its value is either slave, private, or shared.
> >  ```json
> >      "rootfsPropagation": "slave",
> >  ```
> > +
> > +## No new privileges
> > +
> > +Setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
>
> nope
> … <#1443586824_>
> On Wed, Jan 20, 2016 at 5:11 PM Vish Kannan <notifications@github.com>
> wrote: In runtime-config-linux.md <#290 (comment)
> <https://github.com/opencontainers/specs/pull/290#discussion_r50328023>>:
> > @@ -503,3 +503,14 @@ Its value is either slave, private, or shared. >
> ```json > "rootfsPropagation": "slave", > ``` > + > +## No new privileges >
> + > +Setting `noNewPrivileges` to true prevents the processes in the
> container from gaining additional privileges. Should the default be true ?
> — Reply to this email directly or view it on GitHub <
> https://github.com/opencontainers/specs/pull/290/files#r50328023>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/290/files#r50328465>.
>